tests/functional/aws-node-sdk/test/legacy/testBucketVersioning.js

```diff
@@ -0,0 +1,131 @@
+import assert from 'assert';
```

Comment:
I'm not very familiar with the organization of the tests here.
Where do you think it should go?